### DISCUSSION TITLE
Request for all future YouTube commits to include what desktop_polymer.js ID

### DISCUSSION THREAD

**Discussion by y2k04** [uBlockOrigin/uAssets#26568] [General]
I rebooted [does-uBO-bypass-yt](https://github.com/y2k04/does-uBO-bypass-yt) but I have the issue of "Where am I going to get the uBlock Origin update IDs?".

The temporary YouTube fix commit does work, you just have to clear the cache. So, I created a pull request in [stephenhawk8054/misc](/stephenhawk8054/misc/pull/34) to add c01ea7e3 to the list of IDs that are fixed. However, the pull request was closed with this comment:
![image](https://github.com/user-attachments/assets/68cfacb8-8931-4efb-8a9c-4ea314844a43)

Instead of using a repository to host a single file with every single ID that has been patched, why not add the patched ID to the commit description or title so that it can be fetched using api.github.com?

Any thoughts?

**Comment by y2k04** [comment]
![image](https://github.com/user-attachments/assets/820012de-b3fb-42c5-838c-15ac9277b069)

**Comment by stephenhawk8054** [comment]
YT does not actively circumvent uBO with anti-adblock any more.

**Reply by y2k04** [reply]
> YT does not actively circumvent uBO with anti-adblock any more.

https://github.com/uBlockOrigin/uAssets/commit/d1352aaee8bd10fdfd88b8693d91cc9bcbde9979
https://community.spiceworks.com/t/youtubes-war-with-adblockers-continues/1156074 <- I got this popup this morning

**Reply by stephenhawk8054** [reply]
I said

> not **actively** circumvent uBO with anti-adblock any more.

When was the last time you see they updated the new anti-adblock? There was an issue with the syntax of a previous commit and made the filter matches their baits, and we updated the filter again. It's NOT because they updated the anti-adblock. Again, they don't actively update their anti-adblock any more. There's no points to watch every youtube ID any more. We don't have enough free time for that.

Just because today you see the anti-adblock does not mean they are actively circumventing uBO with anti-adblock.

**Reply by y2k04** [reply]
I'm still gonna keep it up though. I have time to maintain repos.

**Reply by stephenhawk8054** [reply]
Feel free to do that yourself, but please don't involve us and we won't do that any more because as I said, they DON'T actively do that any more. Updating the ID is not just looking at the ID and write it to the file. We need to read their code to see if there's any changes. And as I said, there are no changes with the anti-adblock code paths.
